lib/smile-identity-core/id_api.rb

```diff
@@ -98,21 +96,29 @@ def setup_requests
       request.run
     end
 
-    def configure_json
-      body = {
-        timestamp: @timestamp,
-        sec_key: determine_sec_key,
-        partner_id: @partner_id,
-        partner_params: @partner_params
-      }
-
-      body.merge!(@id_info)
-      JSON.generate(body)
+    def id_verification_request
```

Comment:
I think we should revert the function names. The function names in this library match the function names in all the other libraries. This is by design, so when someone is looking to implement the same functionality in a different library, we are all using the same verbiage.

Reply:
This comment applies throughout.

Reply:
Changed back in f5333479b3d640a2d292f42ef1da107f8ec4e83b